### DISCUSSION TITLE
Automerged enabled PRs are gated due to "required review" policy

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#24122] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Auto-merged enabled PRs are merged with no review

### What you tried so far.

Googled

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
If you're running your own got on github.com, you could then run your own approve bot too: https://github.com/renovatebot/renovate-approve-bot

**Reply by rarkins** [reply]
Or potentially a github action with its own app or token which does the same

**Reply by lmilbaum** [reply]
@rarkins Where can I find instructions on setting it up?

**Reply by rarkins** [reply]
Looks like we don't have any, but the code is very simple: https://github.com/renovatebot/renovate-approve-bot/blob/main/index.js
